task2/script.js

```diff
@@ -0,0 +1,17 @@
+const liElement = document.createElement('li');
+
+const anchorElement = document.createElement('a');
+anchorElement.setAttribute('href', '1.html');
+anchorElement.textContent = 'これです';
+
+const bookmarkImg = document.createElement('img');
+bookmarkImg.setAttribute('src', 'bookmark.png');
+bookmarkImg.setAttribute('alt', 'ブックマーク');
+
+anchorElement.appendChild(bookmarkImg);
```

Comment:
【MUST】

Now it's in the browser.
`これです+ img`
the intention of this assignment is
`img + これです`
I think it should be output like this.

- So try to use insertAdjacentElement instead of appendChild😆
[reference](https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentElement)

Reply:
I learned about "insertAdjacentElement".
I would like to change it that way.

Thank you for the review.

Reply:
`a.insertAdjacentElement('afterbegin', img);`
good! Thanks for the fix.

`li.insertAdjacentElement('afterbegin', a)`
I think you can leave this one as the original appendChild.
I'm sorry I confused you.
`ul + li + a **(here!)** 'これです'`
↑I thought I should use .insertAdjacentElement to put the img in the **(here!)** part.

If you have any questions, please feel free to ask in Japanese on Discord.
I'm sorry it's hard to understand in English.